src/App.tsx

```diff
@@ -1,61 +1,110 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState('0');
+  const [isClicked, setIsClicked] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+
+  const clearForm = () => {
+    setUser('0');
+    setTitle('');
+    setIsClicked(false);
+  };
+
+  const handleSubmit
+   = (event:React.SyntheticEvent) => {
+     event.preventDefault();
+     setIsClicked(true);
+
+     if (!title || user === '0') {
+       return;
+     }
+
+     const id: number = todosFromServer.sort((a, b) => b.id - a.id)[0].id + 1;
```

Comment:
Consider using a more descriptive variable name instead of `id`. For example, `newTodoId`. 🤔